suite-common/wallet-utils/src/tokenUtils.ts

```diff
@@ -39,3 +39,6 @@ export const enhanceTokensWithRates = (tokens: Account['tokens'], coins: CoinFia
 
     return tokensWithRates;
 };
+
+export const shortTokenSymbol = (symbol: string): string =>
```

Comment:
It is better to use css shortening so that when user copies the text, the full symbol name is copied. In case of this `...` are copied.

We use it already for NFT names in `FormattedNftAmount`. Probably it would be better to replace `max-width: 145px;` by `max-width: 10ch`